libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/NavigationPerformanceMetadata.java

```diff
@@ -0,0 +1,54 @@
+package com.mapbox.services.android.navigation.v5.navigation;
+
+import android.app.ActivityManager;
+import android.content.Context;
+import android.os.Build;
+import android.util.DisplayMetrics;
+
+class NavigationPerformanceMetadata {
+  private static final String OPERATING_SYSTEM = "Android - " + Build.VERSION.RELEASE;
+  private static final String DEVICE = Build.DEVICE;
+  private static final String MANUFACTURER = Build.MANUFACTURER;
+  private static final String BRAND = Build.BRAND;
+  private static final String ABI = Build.CPU_ABI;
+  private static final String VERSION = String.valueOf(Build.VERSION.SDK_INT);
+
+  private final String version;
+  private final String screenSize;
+  private final String country;
+  private final String device;
+  private final String abi;
+  private final String brand;
+  private final String ram;
+  private final String os;
+  private final String gpu;
+  private final String manufacturer;
+
+  NavigationPerformanceMetadata(Context context) {
+    this.version = VERSION;
+    this.screenSize = getScreenSize(context);
+    this.country = getCountry(context);
```

Comment:
> Is the `Context` necessary here?

What about passing in the country through the constructor so the `Context` wouldn't be necessary?